package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clarity-js",
-  "version": "0.1.0",
+  "version": "0.1.1",
   "description": "An analytics library that uses web page interactions to generate aggregated insights",
```

Comment:
Should we start to use the same version for the package and the schema encoded version?